### PR TITLE
RA-1522: Fixed radio buttons on add new condition UI

### PR DESCRIPTION
**Pull request by haripriya999**
# RA-1522: Fixed radio buttons on add new condition UI

## Description of what I changed
Changed spelling of Active and Inactive status and removed clear option

## Issue I worked on

see https://issues.openmrs.org/browse/RA-1522

## Checklist: I completed these to help reviewers :)

- [x] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

  No? -> [read here](https://wiki.openmrs.org/display/docs/Pull+Request+Tips) on how to squash multiple commits into one

- [x] My IDE is configured to follow the [**code style**](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.

  No? Unsure? -> [configure your IDE](https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE), format the code and add the changes with `git add . && git commit --amend`

- [x] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`

- [x] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.

  No? -> execute above command

- [x] All new and existing **tests passed**.

  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.

- [x] My pull request is **based on the latest changes** of the master branch.

  No? Unsure? -> execute command `git pull --rebase upstream master`

### CODE REVIEW

File: omod/src/main/webapp/resources/scripts/conditionlist/controllers/addcondition.controller.js [hunk 1]
```diff
@@ -51,10 +51,6 @@ function ConditionController($scope, RestfulService, ConditionModel, ConceptMode
             self.saveCondition();
         }
 
-    self.unselectStatus = self.unselectStatus || function () {
```
Comment (dkayiwa) [inline omod/src/main/webapp/resources/scripts/conditionlist/controllers/addcondition.controller.js]: Is this part of the ticket?

File: omod/src/main/webapp/resources/scripts/conditionlist/controllers/addcondition.controller.js [hunk 2]
```diff
@@ -65,5 +61,4 @@ function ConditionController($scope, RestfulService, ConditionModel, ConceptMode
 
     // bind functions to scope
     $scope.validateCondition = self.validateCondition;
-    $scope.unselectStatus = self.unselectStatus;
```
Comment (dkayiwa) [inline omod/src/main/webapp/resources/scripts/conditionlist/controllers/addcondition.controller.js]: Is this part of the ticket?

File: omod/src/main/webapp/pages/conditionlist/addCondition.gsp [hunk 3]
```diff
@@ -72,7 +72,6 @@
             <input type="radio" id="status-2" class="condition-status" value="${ui.message('coreapps.conditionui.inactive.label')}" name="status" ng-model="condition.status"/>
             <label for="status-2">${ui.message('coreapps.conditionui.inactive.label')}</label>
         </p>
-        <i class="icon-remove delete-item" title="${ui.message("general.clear")}" ng-click="unselectStatus()"></i>
```
Comment (dkayiwa) [inline omod/src/main/webapp/pages/conditionlist/addCondition.gsp]: Is this part of the ticket?

File: omod/src/main/webapp/resources/scripts/conditionlist/controllers/addcondition.controller.js (hunk 1 above)
Comment (haripriya999) [reply omod/src/main/webapp/resources/scripts/conditionlist/controllers/addcondition.controller.js]: Yes, This is to remove the functionality of cross button 'x'  as mentioned on the ticket

File: omod/src/main/webapp/resources/scripts/conditionlist/controllers/addcondition.controller.js (hunk 2 above)
Comment (haripriya999) [reply omod/src/main/webapp/resources/scripts/conditionlist/controllers/addcondition.controller.js]: yes, to remove the functionality of cross button 'x'

File: omod/src/main/webapp/pages/conditionlist/addCondition.gsp (hunk 3 above)
Comment (haripriya999) [reply omod/src/main/webapp/pages/conditionlist/addCondition.gsp]: yes, to remove the functionality of cross button 'x'

**Pull request merged**